python/ray/actor.py

```diff
@@ -1817,3 +1914,42 @@ def exit_actor():
             f"{worker.mode}. Call this API inside an actor methods"
             "if you'd like to exit the actor gracefully."
         )
+
+
+def _handle_tensor_transport(dst_actor, args):
+    """Handle tensor transport between actors.
+
+    Args:
+        dst_actor: The target actor to receive tensors
+        args: List of arguments that may contain ObjectRefs
+    """
+    from ray.experimental.channel import ChannelContext
+
+    ctx = ChannelContext.get_current()
+
+    # TODO(kevin85421): Currently, GPU objects only support 1 communicator.
+    if len(ctx.communicators) != 1 or not args:
+        return
+
+    actor_id_to_rank = {}
```

Comment:
Nit: Initialize the dictionary here instead of in the middle of the loop. We don't expect the dictionary to change since there is only one communicator anyway.

Reply:
`actor_id_to_rank` is not always needed, which is why I initialize it lazily to avoid unnecessary operations. For example, all arguments are not `ObjectRef`. Let me know your preference.